brainstormforce/ultimate-addons-for-gutenberg
# Fix: Block padding issue.

### Description

- https://brainstormforce.slack.com/archives/CAQ2TSA6A/p1637076864070100

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

Case - 1 - Backward compatibility check.
Case - 2 - No Debug log
Case - 3
- Drag and drop TOC.
- Set padding empty it will inherit 30px
- set padding 0 it will add padding 0 in frontend as well as backend.


### Checklist:
